tests/__snapshots__/test_help_messages.ambr

```diff
@@ -28,6 +59,35 @@
   ╰──────────────────────────────────────────────────────────────────────────────╯
   
   
+  '''
```

Comment:
Let's add test for something else than top level, something  like `snow connection` just to be sure

Reply:
I've added tests for all commands without subcomamnds.
These were skipped due to `list` `add` commands do not display help but execute action.